docs/api/connectivity/bluetooth/BLE.md

```diff
@@ -47,7 +47,7 @@ the events (methods) you are interested in, the others will inherit the do-nothi
 
 All BLE operations are executed on an instance of BLE accessible through a function in the header `ble/BLE.h`.
 
-```c
+```cpp
```

Comment:
The fixes in .md files should be separate commit to understand what they are fixing

Reply:
I agree a separate commit for the docs changes (even if they are to code samples) would be clearer. I will add comments here anyway.

Reply:
Fixed